packages/docs/docs/tutorial/part1.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 1
+---
+
+# Tutorial Part 1 - The basics
+
+In this tutorial, we're going to introduce the basic concepts of AI.JSX one step at
+a time. All of the code for these tutorials can be found on GitHub at
+https://github.com/fixie-ai/ai-jsx/tree/main/packages/tutorial.
+
+Let's start with the basic "hello world" example for AI.JSX,
+which invokes a Large Language Model with a fixed prompt, and
+prints the result to the console. Here is the complete
+application:
+
+```tsx filename="packages/tutorial/src/part1.tsx"
+import * as LLMx from 'ai-jsx';
+import { ChatCompletion, UserMessage } from 'ai-jsx/core/completion';
+
+const app = (
+  <ChatCompletion>
+    <UserMessage>Write a Shakespearean sonnet about AI models.</UserMessage>
+  </ChatCompletion>
+);
+
+const renderContext = LLMx.createRenderContext();
+const response = await renderContext.render(app);
+console.log(response);
+```
+
+You can run this yourself from the AI.JSX tree by running:
+
+```
+yarn workspace tutorial run part1
+```
+
+## How it works
+
+The first part of the code defines the variable `app`, which
+consists of a JSX component tree that will be rendered by the
+AI.JSX runtime. This is very similar to React, which uses
+JSX components to generate a DOM tree. Here, we are using JSX
+to describe the structure of the LLM model invocations and prompts
+in a compact, declarative way.
+
+The `<ChatCompletion>` component is the root of the tree, and is responsible
+for generating a single invocation of the LLM -- in this case, OpenAI's
```

Comment:
Uber-nit: let's use an em-dash (–) rather than two hyphens (--). You can produce one on mac by hitting `alt`+`-`.